src/live.html

```diff
@@ -130,28 +130,34 @@ <h1 id="title">Live</h1>
 		<nav>
 			<ul>
 				<li data-bind-view="dayof">
-					<a href="#/dayof">Home</a>
+					<a href="#/dayof">Day of</a>
+				</li>
+				<li data-bind-view="live">
+					<a href="#/live">Live</a>
 				</li>
 				<li data-bind-view="schedule">
 					<a href="#/schedule">Schedule</a>
 				</li>
-				<li data-bind-view="travel">
-					<a href="#/travel">Travel</a>
-				</li>
 				<li>
 					<a href="http://maps.hackupc.com/#/UPC" target="_blank">Maps</a>
 				</li>
+				<li>
+					<a href="https://mentors.hackupc.com/" target="_blank">Mentors</a>
+				</li>
 				<li data-bind-view="hardware">
-					<a href="#/hardware">Hardware</a>
+					<a href="#/hardware">Hardware List</a>
 				</li>
 				<li data-bind-view="challenges">
-					<a href="#/challenges">Challenges</a>
+					<a href="#/challenges">Challenges & Prizes</a>
+				</li>
+				<li data-bind-view="travel">
+					<a href="#/travel">Travel Info</a>
 				</li>
 				<li data-bind-view="rules">
 					<a href="#/rules">Rules</a>
 				</li>
 				<li data-bind-view="faq">
-					<a href="#/faq">FAQ</a>
+					<a href="#/faq">FAQs</a>
```

Comment:
`FAQ` was fine.
`FAQ's` or `FAQs` would only be proper if there were multiple FAQ sections

Reply:
I added the 's' because I've seen both versions online and it looks better to me. (for just one FAQ section)